ScrollEdgeControl/Core/ScrollStickyVerticalHeaderView.swift

```diff
@@ -143,6 +143,13 @@ public final class ScrollStickyVerticalHeaderView: UIView {
   public override func didMoveToSuperview() {
 
     super.didMoveToSuperview()
+    
+    // We cannot rely on the existence of self.superview to decide whether to setup scrollView,
+    // because it did not exist yet on iOS 13.
+    guard componentState.hasAttachedToScrollView == false else {
+      // No need to setup scrollView.
+      return
+    }
```

Comment:
needs this check otherwise assertionFailure will occur in setupInScrollView (`\(self) is alread atttached to scrollView\(targetScrollView as Any)`)